### PR TITLE
Add Spree.solidus_version and remove SOLIDUS_VERSION

### PR DESCRIPTION
**Pull request by jhawthorn**
Interested in thoughts on this. We definitely need should have a `solidus_version` method, and I don't see a reason to have both that and a `SOLIDUS_VERSION` file.

I wish we'd added this method in the 1.0.0 release, but I think this is better late than never.

### CODE REVIEW

File: api/solidus_api.gemspec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- encoding: utf-8 -*-
-version = File.read(File.expand_path("../../SOLIDUS_VERSION", __FILE__)).strip
+require File.expand_path('../../core/lib/spree/core/version.rb', __FILE__)
```
Comment (tvdeyen) [inline api/solidus_api.gemspec]: Come on, its 2015/16 `require_relative` is a thing ;)

**Comment by tvdeyen** [comment]
Otherwise 👍

File: api/solidus_api.gemspec (hunk 1 above)
Comment (jhawthorn) [inline api/solidus_api.gemspec]: Fixed, thanks.

**Comment by cbrunsdon** [comment]
Yea, all good by me, thanks @jhawthorn :+1:

**Comment by jordan-brough** [comment]
:+1:

**Pull request merged**